### PR TITLE
fix: Fix overlapping columns in event dashboard

### LINKED ISSUE #3477: Event Dashboard: overlapping columns in `Tickets/Attendees`
**Describe the bug**

Overlapping columns in `Tickets/Attendees`

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to 'Events/Ticket/Attendee'
2. Try to resize
3. See error

**Screenshots**

Screenshot 2019-08-25 at 10 26 34 PM

### PR DESCRIPTION
**Pull request by amreshk005**
Fixes #3477 

#### Short description of what this resolves:

- Now the overlapping problem is solved in the Event Dashboard.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

Screen Shot 2019-09-10 at 7 19 51 PM

### CODE REVIEW

**Comment by amreshk005** [comment]
@mrsaicharan1 I have added the screenshot. 

Please check it out.

**Comment by amreshk005** [comment]
@mrsaicharan1 I have modified the commit.

Please look over it.

File: app/controllers/events/view/tickets/attendees/list.js [hunk 1]
```diff
@@ -10,35 +10,43 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
...
-        valuePath : 'firstname'
+        valuePath : 'firstname',
+        width     : 100
       },
       {
         name      : 'Last Name',
-        valuePath : 'lastname'
+        valuePath : 'lastname',
+        width     : 90
+
```
Comment (kushthedude) [inline app/controllers/events/view/tickets/attendees/list.js]: Remove this extra space

Comment (amreshk005) [reply app/controllers/events/view/tickets/attendees/list.js]: @kushthedude I have fixed the extra spaced

Please check it out.

**Comment by amreshk005** [comment]
@kushthedude I have fixed the extra spaced

Please check it out.

**Comment by kushthedude** [comment]
> @kushthedude I have fixed the extra spaced
> 
> Please check it out.

Give screenshot in the smallest screen size or mobile screen to check the overlap.

**Pull request merged**